test-cargo-miri/run-test.py

```diff
@@ -21,6 +21,9 @@ def cargo_miri(cmd):
         args += ["--target", os.environ['MIRI_TEST_TARGET']]
     return args
 
+def scrub_timing_info(str):
+    return re.sub("finished in \d+\.\d\ds", "", str)
+
```

Comment:
This is a bit awkward, but I don't think I can stop rustdoc from emitting timing information, so this is the only way to stop fickle test failures.

Reply:
That seems fair, but the usual approach we take for variable output like this is slightly different: I suggest you make a `normalize_stdout` function which replaces the timing info by something like `finished in $TIME`, and you apply that *only to the process output*. The reference file is expected to contain the already-normalized output.